packages/core/src/Stores/client-store.js

```diff
@@ -406,6 +406,7 @@ export default class ClientStore extends BaseStore {
             setPrevRealAccountLoginid: action.bound,
             setP2pAdvertiserInfo: action.bound,
             setPrevAccountType: action.bound,
+            getResidence: action.bound,
```

Comment:
getResidence is not an action but computed value. right?

Reply:
Removed it